### ISSUE TITLE
Add examples of requests to documentation of REST API

### ISSUE THREAD

**Issue by kivio** [cloudera/cm_api#60]
Add examples or move documentation to swagger ui - currently is pretty hard to recognize how request that i want to do should look, also error responses are not informative

**Comment by tritab** [comment]
Good idea with the swagger ui. In the meantime, you might find the python examples helpful:
- https://github.com/cloudera/cm_api/tree/master/python/examples
